### ISSUE TITLE
writeLP and writeMPS lost with JuMP 0.19

### ISSUE THREAD

**Issue by dourouc05** [JuliaOpt/JuMP.jl#1907]
I'm a long-time user of the `writeLP` function to ease model debugging (especially when a problem happens deep inside a loop, where getting access to the model from the REPL is not always easy — and it is so convenient to load a LP file in CPLEX interactive or Gurobi shell!). However, when migrating to JuMP 0.19, this function (along with `writeMPS`) seems to have disappeared. I could not see any trace of it in the documentation (nor a related title), in NEWS, in MathOptInterface, in LinQuadOptInterface, so I feel a bit lost… Has this functionality been deprecated or moved to another package? I guess it should be documented (`@deprecate`).

**Comment by odow** [comment]
See https://discourse.julialang.org/t/writemps/21669/4

https://github.com/odow/MathOptFormat.jl

**Comment by joaquimg** [comment]
Should we move to juliOpt, tag and add to news?

**Comment by odow** [comment]
Probably. It's a good candidate to join the new JuMP-dev organisation. I also want to rename it to `MathOptFormats.jl`.

**Comment by dourouc05** [comment]
What about providing an error message for the old functions that point to this new package? 

```
writeLP(x…) = error("writeLP is no more provided by JuMP; instead, you can check MathOptFormat.jl")
writeMPS(x…) = error("writeMPS is no more provided by JuMP; instead, you can check MathOptFormat.jl")
```

**Comment by odow** [comment]
Sure, we would accept a PR for this. If MathOptFormats.jl gets tagged, we will probably re-introduce this functionality to JuMP via a package dependency.

**Comment by mlubin** [comment]
Fixed by #1910
